MSAL/NativeAuthEndToEndTestPlan.xctestplan

```diff
@@ -62,6 +62,8 @@
     }
```

Comment:
I think the Mock API test plan configuration should be removed

Reply:
No, it should not. I already created a new schema dedicated to E2E tests in the branch where I want to run those tests in the pipeline. I can do this change here too